shippy-measurement/analysis/analysis.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs');
+const path = require('path');
```

Comment:
Will this also all be globals on `window`? Just wondering because there's no function-scope module as is the case with the other `shippy-*.js` files.